site/content/docs/user/ingress.md

```diff
@@ -133,6 +134,40 @@ kubectl wait --namespace ingress-nginx \
 
 Refer [Using Ingress](#using-ingress) for a basic example usage.
 
+### Kong
+
+To run Kong ingress, create a ClusterRoleBinding granting full cluster
+admin privileges to the `default` ServiceAccount in `kube-system`:
+
+```bash
+kubectl create clusterrolebinding add-on-cluster-admin \
+  --clusterrole=cluster-admin \
+  --serviceaccount=kube-system:default
+```
+
+Next, install the [Kong helm chart](https://github.com/Kong/charts),
+setting the following values:
+
+```yaml
+proxy:
+  type: NodePort
```

Comment:
is this NodePort as in Service.NodePort?

Reply:
Yes.

Reply:
a NodePort by default only listen on ports `30000 - 32767`, do you have it working this way?